SRM 325/RGBStreet.cpp

```diff
@@ -0,0 +1,196 @@
+#include <cstdio>
+#include <cmath>
+#include <cstring>
+#include <ctime>
+#include <iostream>
+#include <algorithm>
+#include <set>
+#include <vector>
+#include <sstream>
+#include <typeinfo>
+#include <fstream>
+
+using namespace std;
+
+int dp[100];
+
+class RGBStreet {
+    public:
+
+    int cvt_strto_int(string s) {
+        int numb = 0;
+        int n =  s.length();
+        for(int i=0; i<n; ++i) {
+            numb = numb*10 + (s[i]-'0');
+        }
+        return numb;
+    }
+
+    vector<vector<int> > extractPrices(vector<string> houses) {
```

Comment:
Here you have used `extractPrices` which is in camel case and above you have used  `cvt_strto_int ` in snake case.